ppcomp/ppcomp.py

```diff
@@ -277,7 +276,7 @@ def remove_thought_breaks(self):
     def suppress_footnote_tags(self):
         """Remove footnote tags"""
         if self.args.ignore_format or self.args.suppress_footnote_tags:
-            self.text = re.sub(r"[\[]*Footnote ([\d\w]+):\s([^]]*?)[\]]*", r"\1 \2", self.text,
+            self.text = re.sub(r"[\[]*Footnote ([\d\w]+):\s([^]]*?)[]]*", r"\1 \2", self.text,

```

Comment:
Unless this is broken now I would suggest leaving the `\` in to help indicate that the following `]` is a literal and not the end of a character class.

Reply:
Doesn't seem to matter, I just changed it because Pycharm code inspector warns "Redundant character escape '\]' in RegExp"